### PR TITLE
+Send Direct Message

### PR DESCRIPTION
**Pull request by gigihiski**

### CODE REVIEW

File: src/client/slack_client.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { WebClient, WebAPICallResult, WebAPICallError } from "@slack/client"
+import { WebClient, RTMClient, WebAPICallResult, WebAPICallError } from "@slack/client"
```
Comment (wahyusumartha) [inline src/client/slack_client.ts]: RTMClient is never used in this class, 
I think we could remove it @gigihiski

**Review by wahyusumartha — CHANGES_REQUESTED** [review]

File: src/__tests__/slack_message_config.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { Configurator } from "../config/slack_message_config"
+
+describe("Slack Message Configuration", () => {
```
Comment (wahyusumartha) [inline src/__tests__/slack_message_config.spec.ts:3]: thanks for adding unit test 👍

File: src/client/slack_client.ts (hunk 1 above)
Comment (gigihiski) [reply src/client/slack_client.ts]: alright mas @wahyusumartha i think we can remove it

File: src/__tests__/slack_message_config.spec.ts (hunk 2 above)
Comment (gigihiski) [reply src/__tests__/slack_message_config.spec.ts:3]: no problem :)

File: src/helper/post_message_helper.ts [hunk 3]
```diff
@@ -13,19 +13,20 @@ const PostMessageHelper = async (
...
 	callback: Callback
 ) => {
-	const configurator = new Configurator()
-	const token = configurator.getSlackConfig().token
+	const configurator = new Configurator("env.yml")
+	const authToken = configurator.getSlackConfig().authToken
 	const channel = configurator.getSlackConfig().channel
 	const channelName = configurator.getSlackConfig().channelName
-	const client = new SlackClient(token)
+	const client = new SlackClient(authToken)
```
Comment (wahyusumartha) [inline src/helper/post_message_helper.ts:20]: we should have two SlackClient, 
One SlackClient to retrieve channel info , 
Another SlackClient with both authentication token to open conversation and send a message

**Review by wahyusumartha — CHANGES_REQUESTED** [review]

File: src/helper/post_message_helper.ts (hunk 3 above)
Comment (gigihiski) [reply src/helper/post_message_helper.ts:20]: ok mas will create the other SlackClient and create open connection function

**Pull request closed without merging**